src/styles/block/main-section.scss

```diff
@@ -0,0 +1,61 @@
+.main-section {
+  display: grid;
+  row-gap: 92px;
+  padding-block: 60px;
+  background-color: $nav-font-color;
+
+  @include inline-padding;
+
+  @include on-tablet {
+    padding-inline: 72px;
+    padding-block: 120px;
+  }
+
+  @include on-pc {
+    padding-inline: 120px;
+    padding-block: 120px;
+    row-gap: 160px;
+  }
+}
+
+.about-us {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+
+  @include on-tablet {
+    flex-direction: row;
+    text-align: left;
+    justify-content: space-between;
+  }
+
+  @include on-pc {
+    align-items: center;
+    justify-content: center;
+    column-gap: 130px;
+  }
+}
+
+.compare-bikes {
+  display: flex;
+  flex-direction: column;
+  row-gap: 40px;
+  margin: 0 auto;
```

Comment:
Technically BEM blocks should not have margins, but IMO `margin: 0` is still not so critical, please remove at least `margin: auto` from the block

You should use it on an element